src/zenml/zen_server/routers/steps_endpoints.py

```diff
@@ -265,6 +269,10 @@ def get_step_logs(
     artifact_store = _load_artifact_store(logs.artifact_store_id, store)
     return str(
         _load_file_from_artifact_store(
-            logs.uri, artifact_store=artifact_store, mode="r"
-        )
+            logs.uri,
+            artifact_store=artifact_store,
+            mode="rb",
+            offset=offset,
+            length=length,
+        ).decode()
```

Comment:
why did you add the extra `decode` here ?

Reply:
I had to change the `mode` to binary reads, as the seeking is only available for the cloud filesystems when using this mode.